src/vogleditor/vogleditor.cpp

```diff
@@ -471,7 +471,7 @@ VoglEditor::Prompt_Result VoglEditor::prompt_generate_trace()
 #endif
 
             QString cmdLine = m_pLaunchTracerDialog->get_command_line();
-            QProcessEnvironment env = m_pLaunchTracerDialog->get_process_environment();
+            QProcessEnvironment env = m_pLaunchTracerDialog->get_process_environment (sizeof (void *) < 8);
```

Comment:
The true answer to this question, IMO, is figuring out how to build a separate, 32-bit tracer using Nix.